### ISSUE TITLE
Collapse component and keyboard navigation

### ISSUE THREAD

**Issue by laurent-cintas** [Orange-OpenSource/Orange-Boosted-Bootstrap#154]
https://boosted.orange.com/docs/4.2/components/collapse/

With a keyboard navigation, the toggle button appears to be functional only when it is built with an A tag or a BUTTON tag.

But the “Accessibility” paragraph states that « If the control element’s HTML element is not a button (e.g., an `<a>` or `div`>), the attribute role="button" should be added to the element”

https://boosted.orange.com/docs/4.2/components/collapse/#accessibility

Even if we add a role=”Button” (and a tabindex=”0”), onto a DIV element, it's impossible to control the opening and closing of a panel from the keyboard navigation ("enter" key).

- Operating system and version (Windows 10)
- Browser and version (Chrome, Firefox ESR, IE11)
